.github/workflows/ci.yaml

```diff
@@ -249,12 +249,47 @@ jobs:
           push: true
           archive: false
           comment: false
+  push-no-modules:
```

Comment:
I think we need to either rename this (and all the other jobs) to make it clear they are e2e tests, or move these tests to their own workflow file. Probably choosing names like `test-push-no-modules` would be fine.

But also, I think the name should be something more like `test-push-no-module-name`

Reply:
All of the CI jobs in the workflow are e2e tests.

Reply:
Moved to `test-` prefix.